framework/src/governance/lib/datazone/datazone-msk-asset-type-props.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { RemovalPolicy } from 'aws-cdk-lib';
+import { DataZoneCustomAssetTypeFactory } from './datazone-custom-asset-type-factory';
+
+/**
+ * The properties for the DataZoneMskAssetType construct
+ */
+export interface DataZoneMskAssetTypeProps {
+  /**
+   * The DataZone domain identifier
+   */
+  readonly domainId: string;
+  /**
+   * The project identifier owner of the custom asset type
+   * @default - A new project called MskGovernance is created
+   */
+  readonly projectId?: string;
```

Comment:
What happen if this is called twice with  `MskGovernance` passed?

Reply:
the project name is specific to the asset type. Deployment would fail if you create the asset type twice but it doesn't make sense to have it twice. In any case the custom asset type creation would fail because it already exists whatever project you are using to own it.